build.sh

```diff
@@ -319,15 +321,16 @@ download_common() {
 
     if [ "$download_type" == "core" ]; then
         version=$REALM_CORE_VERSION
-        url="https://static.realm.io/downloads/core/realm-core-${version}.tar.xz"
+        url="${REALM_BASE_URL}/core/realm-core-${version}.tar.xz"
     elif [ "$download_type" == "sync" ]; then
         version=$REALM_SYNC_VERSION
-        url="https://static.realm.io/downloads/sync/realm-sync-cocoa-${version}.tar.xz"
+        url="${REALM_BASE_URL}/sync/realm-sync-cocoa-${version}.tar.xz"
     else
         echo "Unknown dowload_type: $download_type"
         exit 1
     fi
 
+    echo "LOOK OVER HERE: ${url}"
```

Comment:
Leftover debugging code?

Putting the URL being downloaded into the line under this seems reasonable (i.e. something like `echo "Downloading dependency: ${download_type} ${version} from ${url}"`

Reply:
Ops, sorry, I'll do it